src/components/Portal/Portal.js

```diff
@@ -203,6 +211,9 @@ Portal.propTypes = {
   /** @type {boolean} Focus trap. */
   focusTrap: PropTypes.bool,
 
+  /** Pass any of the options available in https://github.com/focus-trap/focus-trap#focustrap--createfocustrapelement-createoptions */
```

Comment:
The link didn't work, however, this one did:
```suggestion
  /** Pass any of the options available in https://github.com/focus-trap/focus-trap#createfocustrapelement-createoptions */
```